GolfExample_TCAPI/assessmenttemplate.html

```diff
@@ -128,18 +141,23 @@
                     },
                     interactionType: questionType,
                     correctResponsesPattern: [
-                        String(correctAnswer)
+                        strCorrectAnswer
                     ]
                 }
             };
 
+            //format numeric response pattern. 
+            if (qObj.definition.interactionType == "numeric"){
+                qObj.definition.correctResponsesPattern[0] = strCorrectAnswer + ":" + strCorrectAnswer;

```

Comment:
I know there was further discussion around this relatively recently, is this still correct based on that discussion?